custom/enikshay/management/commands/get_users_for_bets.py

```diff
@@ -0,0 +1,73 @@
+import csv
+from django.core.management.base import BaseCommand
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.users.models import CommCareUser
+from corehq.apps.custom_data_fields.models import CustomDataFieldsDefinition
+from corehq.apps.users.views.mobile.custom_data_fields import UserFieldsView
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def confirm(self):
+        return raw_input("Continue?\n(y/n)") == 'y'
+
+    def get_data_fields(self):
+        data_model = CustomDataFieldsDefinition.get_or_create(
+            self.domain,
+            UserFieldsView.field_type,
+        )
+        return sorted(data_model.fields)
+
+    def handle(self, domain, **options):
+        self.domain = domain
+        self.locations_by_id = {
+            loc.location_id: loc for loc in SQLLocation.objects.filter(domain=domain)
+        }
+        filename = 'agency_users.csv'
```

Comment:
won't this get saved in the `current` release directory? Are the permissions set up correctly to be able to scp from there?